robot.py

```diff
@@ -0,0 +1,110 @@
+import asyncio
+from contextlib import asynccontextmanager
+
+from viam.components.board import Board
+from viam.components.servo import Servo
+from viam.robot.client import RobotClient
+from viam.rpc.dial import DialOptions
+
+
+@asynccontextmanager
+async def create_robot(creds, address):
+    """
+    This makes a connection to the hardware, creates a Robot object, and then
+    closes the connection when the context manager exits.
+    """
+    opts = RobotClient.Options(
+        refresh_interval=0,
+        dial_options=DialOptions(credentials=creds)
+    )
+    client = await RobotClient.at_address(address, opts)
+    robot = Robot(client)
+
+    await robot.start()
+    try:
+        yield robot
+    finally:
+        await robot.stop()
+        await client.close()
+
+
+class Robot:
+    UPPER_POSITION = 30
+    LOWER_POSITION = 0
+    WIGGLE_AMOUNT = 5
+    INACTIVITY_PERIOD_S = 5
+
+    def __init__(self, client):
+        """
+        This class is in charge of raising and lowering the robot's hand, and
+        wiggling the hand if it has been raised for too long.
+
+        WARNING: this class is not thread safe!
+
+        The client passed in is a RobotClient object.
+        """
+        self._client = client
```

Comment:
The client is only used to get the servo and the board. Let's just get both of those in the context manager, pass just the servo into this class, and return the board from the context manager without going in here at all!